### ISSUE TITLE
Screenshot carrousel cannot be reached with keyboard only from search results

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/pub-dev#7284]
It doesn't get focus when using tab.

![image](https://github.com/dart-lang/pub-dev/assets/8613953/834a5654-cb98-4149-b021-44a0519712be)

**Comment by sigurdm** [comment]
Also on the package page details sidebar

**Comment by sigurdm** [comment]
Once this is fixed, we should also ensure that the focus goes back to the screenshots button after the carousel is closed.
